### PR TITLE
build(changelog): generate a changelog by using conventional-changelog

### PR DESCRIPTION
**Pull request by antleblanc**
Based on actual commits history:

``` bash
$ ./node_modules/.bin/conventional-changelog -p angular -i CHANGELOG.md -s
```

It's generate the following diff:

``` diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index e69de29bb2d1..d983bacdb70b 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -0,0 +1,29 @@
+<a name="0.0.0"></a>
+# 0.0.0 (2018-09-02)
diff --git a/CHANGELOG.md b/CHANGELOG.md
index e69de29bb2d1..d983bacdb70b 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -0,0 +1,29 @@
+<a name="0.0.0"></a>
+# 0.0.0 (2018-09-02)
+
+
+### Bug Fixes
+
+* **build:** use a separate rule for ESlint ([ccd05e9](https://github.com/jleveugle/ovh-manager/commit/ccd05e9))
+* **eslint:** restrict eslint in development to /packages folder due to symlinks ([2194ff3](https://github.com/jleveugle/ovh-manager/commit/2194ff3))
+* **less:** bootstrap relative path ([b8c83cf](https://github.com/jleveugle/ovh-manager/commit/b8c83cf))
+* **less:** downgrade less to 2.3.1 to be compliant with ovh-ui-kit (temp) ([bbd1516](https://github.com/jleveugle/ovh-manager/commit/bbd1516))
+* **License:** fix typo in license folder ([0c2ea79](https://github.com/jleveugle/ovh-manager/commit/0c2ea79))
+* **ngAnnotate:** add ngInject when needed ([eaa9396](https://github.com/jleveugle/ovh-manager/commit/eaa9396))
+* **package.json:** set node engine ([b60c9aa](https://github.com/jleveugle/ovh-manager/commit/b60c9aa))
+* **sidebar:** improve sidebar width 100% ([6548c98](https://github.com/jleveugle/ovh-manager/commit/6548c98))
+* add .gitattributes file ([beebf8d](https://github.com/jleveugle/ovh-manager/commit/beebf8d))
+* enhance community profile ([0c07223](https://github.com/jleveugle/ovh-manager/commit/0c07223))
+* **sms:** add ovh-angular-pagination-front as dependencies ([77f1c42](https://github.com/jleveugle/ovh-manager/commit/77f1c42))
+* **sms.options:** hook on $onInit through the v4-links component ([c5df78f](https://github.com/jleveugle/ovh-manager/commit/c5df78f))
+* **sms.users.callback:** add missing annotation ([75505d8](https://github.com/jleveugle/ovh-manager/commit/75505d8))
+
+
+### Features
+
+* **navbar:** add empty navbar ([f56e3f8](https://github.com/jleveugle/ovh-manager/commit/f56e3f8))
+* **ngApp:** use ng-strict-di ([e9af649](https://github.com/jleveugle/ovh-manager/commit/e9af649))
+* **welcome:** lazyload this section ([926b3b8](https://github.com/jleveugle/ovh-manager/commit/926b3b8))
+
+
+
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,11 +17,13 @@
     "lint:js": "eslint --fix .",
     "lint:md": "remark -qf .",
     "precommit": "lint-staged",
-    "commitmsg": "commitlint -E GIT_PARAMS"
+    "commitmsg": "commitlint -E GIT_PARAMS",
```
Comment (jleveugle) [inline package.json:20]: GIT_PARAMS or HUSKY_GIT_PARAMS ?

https://github.com/marionebl/commitlint/issues/366

Comment (antleblanc) [reply package.json:20]: Thanks to point this out.

I've simply followed the **Getting started** part of the [README.md](https://github.com/marionebl/commitlint#getting-started).

After some research it seems that both are acceptable related to the comments: https://github.com/marionebl/commitlint/blob/d636811b4537904231c483a05e91aaf96deb7f6e/%40commitlint/cli/src/cli.js#L211-L216.

Comment (antleblanc) [reply package.json:20]: About **husky** and the issue you are mentioned, yes we will have to do the change if we upgrade to the next major version.

Currently we use the version `^0.14.3`.

ref: https://github.com/jleveugle/ovh-manager/blob/sms/package.json#L25

Comment (jleveugle) [reply package.json:20]: Yes, but i think we should use the next major, it's already in RC, and our project is not going in production next week, better to be up-to-date I think.

What do you think about this ?

Comment (antleblanc) [reply package.json:20]: Bonus: Babel 7 is out! 👍

Comment (antleblanc) [reply package.json:20]: Reminder: This pull request is about `conventional-changelog`:) Let's move on!

**Pull request merged**